_maps/shuttles/escape_pod_large.dmm

```diff
@@ -0,0 +1,130 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE 
+"h" = (
+/obj/structure/chair{
+	dir = 4
+	},
+/obj/item/storage/pod{
+	pixel_x = -24
+	},
+/turf/open/floor/mineral/titanium/blue,
+/area/shuttle/pod_1)
+"r" = (
+/turf/open/floor/mineral/titanium/blue,
+/area/shuttle/pod_1)
+"u" = (
+/obj/structure/grille,
+/obj/structure/window/shuttle,
+/turf/open/floor/plating,
+/area/shuttle/pod_1)
+"A" = (
+/obj/structure/chair{
+	dir = 8
+	},
+/obj/item/device/radio/intercom{
+	pixel_x = 30
+	},
+/turf/open/floor/mineral/titanium/blue,
+/area/shuttle/pod_1)
+"B" = (
+/turf/closed/wall/mineral/titanium,
+/area/shuttle/pod_1)
+"C" = (
+/obj/machinery/computer/shuttle/monastery_shuttle,
+/obj/machinery/light{
+	dir = 1
+	},
+/turf/open/floor/mineral/titanium/blue,
+/area/shuttle/pod_1)
+"H" = (
+/obj/structure/chair{
+	dir = 8
+	},
+/turf/open/floor/mineral/titanium/blue,
+/area/shuttle/pod_1)
+"J" = (
+/obj/structure/chair{
+	dir = 4
+	},
+/obj/machinery/status_display{
+	pixel_x = -32
+	},
+/turf/open/floor/mineral/titanium/blue,
+/area/shuttle/pod_1)
+"L" = (
+/obj/structure/chair{
+	dir = 4
+	},
+/turf/open/floor/mineral/titanium/blue,
+/area/shuttle/pod_1)
+"M" = (
+/obj/structure/chair{
+	dir = 8
+	},
+/obj/machinery/status_display{
+	pixel_x = 32
+	},
+/turf/open/floor/mineral/titanium/blue,
+/area/shuttle/pod_1)
+"O" = (
+/obj/machinery/door/airlock/titanium{
+	name = "Shuttle Airlock"
+	},
+/obj/docking_port/mobile{
```

Comment:
/obj/docking_port/mobile/pod?

Reply:
It didn't have that path before originally on the map but yeah it should be fine to switch it to that